playgrounds/nextjs/pages/index.js

```diff
@@ -0,0 +1,31 @@
+/** @jsxImportSource @emotion/react */
```

Comment:
you have configured `importSource` in the Babel config here, so the explicit pragma shouldn't be needed